### DISCUSSION TITLE
Acknowledge Section

### DISCUSSION THREAD

**Discussion by jcubic** [pyodide/pyodide#2924] [Q&A] [answered]
Do you think you can add some kind of Acknowledge section on the page? I'm mostly asking for my project [jQuery Terminal](https://terminal.jcubic.pl/) used for the REPL. The link from the project website would be helpful for my project. I'm not sure how many projects like this you use but it's something that can help those projects as well.

I have a link to old git repo from the example page, but I can link to the official website in exchange. Those kinds of links mostly are for SEO.

**Comment by rth** [comment, marked as answer]
Hi @jcubic,

so we have added a link to jQuery Terminal in the new REPL https://pyodide.org/en/latest/console.html (right top corner) which points to the homepage. This will also be deployed to the stable REPL https://pyodide.org/en/stable/console.html once we make the release in the coming days. I don't have the stats but the REPL is likely one of the most visited pages of the pyodide.org website. 

Let me know if that is sufficient or if you would prefer there be a an additional Acknowledgement section: it could be put under https://pyodide.org/en/stable/project/about.html where we already acknowledge for some infrastructure support.

**Reply by jcubic** [reply]
Awesome thanks. I think that REPL is fine if the link is in HTML and not in JavaScript as part of the greeting. But I'm not sure how Search engines deal with JS. I've heard that Google supports JS, just fine. I will also update the link to your website.

If it's not too much to ask you can add info about the project (and other projects if you have similar) to about.html. It will be better to have it as static HTML. Acknowledge section is very generic you can also put the links to sponsors if you have any. I have a link to BrowserStack that I use for manual testing. And on the Git repo, I have a link to the most important sites that were helping me. Like SO questions that added an important piece of the code.

**Reply by rth** [reply]
The link is [plain HTML](https://github.com/pyodide/pyodide/blob/main/src/templates/console.html#L65). OK, we will add an acknowledgments section.
